src/SDK/Subscriber/SubscribedEvent.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Subscriber;
+
+use function count;
+use OpenTelemetry\SDK\Attributes;
+use OpenTelemetry\SDK\AttributesInterface;
+use OpenTelemetry\SDK\Trace;
+use OpenTelemetry\SDK\Trace\Span;
+
+final class SubscribedEvent implements SubscribedEventInterface
```

Comment:
What does `SubscribedEvent` mean?

Reply:
It's the nomenclature I have used for Event class